src/Psl/DateTime/Era.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\DateTime;
+
+/**
+ * Represents the two eras in the Gregorian and Julian calendars as an enum.
+ *
+ * This enum distinguishes between the Anno Domini (AD) era, denoting years after the birth of Jesus Christ,
+ * and the Before Christ (BC) era, denoting years before the birth of Jesus Christ. It provides a type-safe way
+ * to represent and work with these two divisions of historical time.
+ */
+enum Era: string
+{
+    case AnnoDomini = 'AD';
+    case BeforeChrist = 'BC';
```

Comment:
wasn't sure here, should we use AD/BC or CE/BCE? i personally lean more towards AD/BC.

but again, do we even need `Era` enum? is this even useful? ( ref `Psl\DateTime\DateTimeInterface::getEra()` ).

Reply:
We use AD/BC, but I haven't ever been in a coding situation where this would add any value.
For me, it's not a must to have this : The check is pretty simple `Year > 0` if you need this.
However, its here - we could as well keep it.